{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/common/views.py

```diff
@@ -1,14 +1,15 @@
-{% if cookiecutter.use_graphql == "n" -%}
-from django.shortcuts import render
-from django.template.exceptions import TemplateDoesNotExist
-{% else -%}
+{% if cookiecutter.use_graphql == "y" -%}
 from django.template.response import TemplateResponse
 from django.views.decorators.cache import never_cache
 from django.views.decorators.csrf import ensure_csrf_cookie
+{% else -%}
+from django.shortcuts import render
+from django.template.exceptions import TemplateDoesNotExist
+from rest_framework import status
```

Comment:
Just moving things around to keep it clean. The top had graphql 2nd but the code below had it first. Also making both affirmative checks and consistent quoting.